clients/include/testing_gemm_sweep.hpp

```diff
@@ -1,237 +0,0 @@
-/* ************************************************************************
```

Comment:
testing_gemm_sweep was redundant. It was basically the same as testing_gemm except that it removed timing code, and made incorrect error calculations. It should be removed.